Formula/handbrake.rb

```diff
@@ -1,10 +1,10 @@
 class Handbrake < Formula
   desc "Open-source video transcoder available for Linux, Mac, and Windows"
   homepage "https://handbrake.fr/"
-  url "https://github.com/HandBrake/HandBrake/releases/download/1.3.3/HandBrake-1.3.3-source.tar.bz2"
-  sha256 "218a37d95f48b5e7cf285363d3ab16c314d97627a7a710cab3758902ae877f85"
+  url "https://github.com/HandBrake/HandBrake.git",
```

Comment:
I don't think we want to switch to a Git clone here. Try adding a `depends_on "libbluray"` instead.

Reply:
👍